sdk/eventhub/event-hubs/samples/sendEvents.ts

```diff
@@ -20,14 +20,23 @@ async function main(): Promise<void> {
 
   console.log("Creating and sending a batch of events...");
   try {
-    const batch = await producer.createBatch();
+    // create a batch targeted to a particular partition
+    const batch = await producer.createBatch({
+      partitionId: "0"
```

Comment:
We only have 1 sample for send at the moment. If we only have 1, then we should use the recommended way to send which is **not** sending to particular partition

Reply:
Would the change be to use a partition key? Or just to forgo specifying either?

Reply:
I removed the partitionId (and specified no partitionKey) and also added in a comment explaining why you'd use them.

Reply:
I modified the sample to no longer pass a partitionId (or partitionKey) and added a comment explaining what's happening and when they'd choose to use partitionId/partitionKey.